conf/dubbo-cp.yaml

```diff
@@ -1,4 +1,4 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
+# Licensed to the Apache Software Foundation (ASF) under one or moreshop-use
```

Comment:
Please check if this is an unexpected change.

Reply:
Sorry, I typed it by mistake